{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "2f51a7c9_0d4d1c5f",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 5005
      },
      "writtenOn": "2022-04-13T14:55:09Z",
      "side": 1,
      "message": "Matches the failure count we had before.",
      "revId": "65d87225030d04609b74c8e11394538a11d1dcc4",
      "serverId": "aea13c4a-0b89-3eca-aee9-e193b1b77aa4"
    }
  ]
}